src/app/Main/Body/QueryPage/Results/TableResults/TableResults-Options.component.tsx

```diff
@@ -64,14 +66,25 @@ export function ResultsTableOptionsComponent(props: {
     isSumRowCol: { showRowSum: boolean; showColSum: boolean };
     tableOption: DisplayOptionType;
     onDisplayOptionsChange: (displayOption: string) => void;
-    onShowRowColSum: (name: string, checked: boolean) => void;
+    onShowRowColSum: (isSumRowCol: {
+        showRowSum: boolean;
+        showColSum: boolean;
+    }) => void;
 }): JSX.Element {
+    const [isSumRowCol, setIsSumRowCol] = useState<SumOptions>({

```

Comment:
Reject: should be initialized with props.isSumRowCol

Reply:
Done.